overlays/python-unsupported-platforms.nix

```diff
@@ -0,0 +1,26 @@
+final: prev:
```

Comment:
In the future, we will probably want the same rule for other languages like Go and Rust. I wonder if we should name this rule e.g. `language-builder-unsupported-platforms` to allow easily extending it in the future.

Reply:
Maybe but it seems like it could get messy trying to get the default platform of every builder in one overlay. Perhaps it would make more sense to abstract the shared logic into a `checkDefaultPlatformsForBuilder` method in `lib`?